### PR TITLE
Add vulnfeed tests to cloudbuild.yaml

### PR DESCRIPTION
**Pull request by another-rex**
Adds a run_tests.sh that will run go test on the vulnfeeds directory. Does not run go vet as that is already done by the lint script.

### CODE REVIEW

**Review by andrewpollock — APPROVED** [review]

**Comment by andrewpollock** [comment]
Looks like golang needs to be installed in the Cloud Build environment

File: cloudbuild.yaml [hunk 1]
```diff
@@ -24,6 +24,9 @@ steps:
 - name: 'gcr.io/oss-vdb/ci'
   dir: docker/importer
   args: ['bash', '-ex', 'run_tests.sh']
+- name: 'gcr.io/oss-vdb/ci'
+  dir: vulnfeeds
+  args: ['bash', '-ex', 'run_tests.sh']
```
Comment (oliverchang) [inline cloudbuild.yaml:29]: Can you add this to our makefile as well?

**Review by oliverchang — APPROVED** [review]

**Comment by another-rex** [comment]
Added the install go script to manually install go 1.18

File: Makefile [hunk 2]
```diff
@@ -24,6 +24,9 @@ importer-tests:
 appengine-tests:
 	cd gcp/appengine && ./run_tests.sh
 
+vulnfeed-tests:
```
Comment (oliverchang) [inline Makefile:27]: Please add this to `all-tests`

File: docker/ci/install_go.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Install go on x86_64, don't do anything on ARM.
+set -eux
+
+case $(uname -m) in
+    x86_64)
```
Comment (oliverchang) [inline docker/ci/install_go.sh:22]: You can get rid of the architecture checking here. This was specific to OSS-Fuzz.

**Pull request merged**